rrat/patients/templates/patients/view_patient.html

```diff
@@ -15,4 +15,21 @@
   class="my-8 h-px border-t-0 bg-transparent bg-gradient-to-r from-transparent via-teal-500 to-transparent opacity-25 dark:via-neutral-400" />
 
   {% include "retina_photos/partials/add_new_retina_photo.html" %}
+  <!-- Section to display existing retina photos for the patient -->
+  <h2 class="text-lg font-semibold mt-8">Existing Retina Photos</h2>
+  <div class="image-gallery mt-4">
+      {% for photo in images %}
+      <div class="image-item border rounded-md shadow-sm p-4">
+          <img src="{{ photo.image.url }}" alt="Retina Image" style="width: 256px; height: 256px; object-fit: cover;" />
+          <div class="image-details text-sm">
+              <p><strong>Position:</strong> {{ photo.position|upper}}</p>
+              <p><strong>Date Created:</strong> {{ photo.date_created|date:"F d, Y" }}</p>
+              <p><strong>Status:</strong> {{ photo.status }}</p>
+              <p><strong>Prognosis:</strong> {{ photo.prognosis }}</p>
```

Comment:
For consistency, instead of using the `strong` tag let's use `<span class="font-semibold">`. This way it will match how we did the patient info above it and ensure we don't do any styling based on HTML (all styling should be done via CSS). You can see this on lines 30 - 59 in the patient_page.html